packages/object-lists/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@colonial-collections/object-lists",
+  "version": "0.0.0",
+  "private": true,
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
+  "scripts": {
+    "db:generate": "drizzle-kit generate:mysql",
+    "db:push": "drizzle-kit push:mysql",
+    "db:drop": "drizzle-kit drop",
+    "db:studio": "drizzle-kit studio"
+  },
+  "devDependencies": {
+    "eslint-config-custom": "*",
+    "tsconfig": "*"
+  },
+  "dependencies": {
+    "dotenv": "16.3.1",
```

Comment:
Do we need this as a production dependency? On Vercel the environment variables are loaded automatically?